Dockerfile

```diff
@@ -1,54 +1,65 @@
-FROM debian:jessie
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.3.0
```

Comment:
> X-Pack is definitely part of the official image and has been turned off via env vars. I can remove it if that will help?

Removing it doesn't remove it from the shipped binary blobs because of the layered file system. If we going to try to use their current build I'd say we should just fork from what they have in their Dockerfiles.